ci/docker_tool

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+IMAGE_NAME=lsinteg
```

Comment:
This should be `$branch_specifier` + a time stamp (or something unique)

For PR's it will be the commit hash, and non-pr's will be branch name `refs/heads/master`